ui/components/src/TimeChart/TimeChart.stories.tsx

```diff
@@ -111,11 +110,7 @@ const meta: Meta<typeof TimeChart> = {
     },
   },
   parameters: {
-    happo: {
-      beforeScreenshot: async () => {
-        await waitForStableCanvas('canvas');
-      },
-    },
+    happo: false,
```

Comment:
Disabling for now since that seems to fix it: https://happo.io/a/1009/p/1516/compare/15bcb3722decbb0821dc36ed0b1f0fadbd49b65b/eb6f9e1bbc81b364d48d514df3e8895c6a3c7442?t=deleted

Although I hope it doesn't come back the next time we add a new visual regression test, Happo support's theory was: "it's possible it is only happening on machines with lower memory"

Reply:
Could you please add a comment in here explaining that these are disabled because we're running into some issues what they look like. That way future folks will know it's worth investigating more and hopefully being able to turn it back on.

Reply:
Whoops I had the PR on auto merge so didn’t get a chance, it does seem like something wrong specifically with the TimeChart test even though it was using the wait for canvas util, didn’t try running with Happo locally, never actually done that before but it was happening consistently on PRs in the recent failures on main